tests/copilot_helper/test_command_dns.py

```diff
@@ -207,17 +305,61 @@ def test_check_domain_env_flag(check_aws_conn, check_r53, fakefs):
 
     expected = "Checking file: copilot/manifest.yml\nDomains listed in manifest file\n\nEnvironment: dev => Domain: v2.app.dev.test.1234\n\nHere are your Certificate ARNs:\nDomain: v2.app.dev.test.1234\t => Cert ARN: arn:1234\n"
 
+    # Check calls to check_r53
+    # Check calls to check_and_return_aws_session
```

Comment:
Are these comments something you meant to do?

Reply:
Nope

Reply:
What are they trying to tell me?

Reply:
I cut them from elsewhere in the code where they were no longer useful, and I think I pasted them by accident.

Reply:
Please delete!

Reply:
I did ages ago. :-D

Reply:
Oh, I must have been accidentally not looking at all the commits. It's happened before when folliwing links from the emails.